scripts/jsdeps.py

```diff
@@ -0,0 +1,38 @@
+import json
+import shutil
+from pathlib import Path
+
+DEST_BASE_DIR = "src/fontra/client/third-party/"
```

Comment:
We should probably find the root of the repo explicitly so this script doesn't depend on the repo root being the current working directory.

I tend to use a pattern like this:
```python
repoRoot = pathlib.Path(__file__).resolve().parent.parent
destDir = repoRoot / "src/fontra/client/third-party/"
```

This also avoids calling `Path()` all the time locally.